### PR TITLE
Remove duplicate configuration for SSL

### PR DESCRIPTION
**Pull request by khorben**
Some configuration files for Apache define SSL certificates twice. This
is not supported and breaks the deployment, as Apache will not start.

### CODE REVIEW

**Comment by quartje** [comment]
This double configuration is there for a reason, there are two conditions in the Apache configuration files for using TLS:

1)    {% if haproxy_backend_tls %}

This TLS configuration is used when the application is behind a  haproxy loadbalancer, and needs to use TLS to encrypt the traffic between the loadbalancer and Apache. 
It can be enabled by setting this variable to true in the group_vars

2    {% if apache_app_listen_address.all is defined %}
 
This variable is defined when no loadbalancer is used and Apache is directly connected to the internet. See chapter minimal installation feature on https://github.com/OpenConext/OpenConext-deploy/wiki/Installation-steps-to-deploy-OpenConext-on-a-single-system-other-than-the-Vagrant-VM-centOS7 for more info.

Bottom line is that both variables should not be enabled at the same time. 

Regards,
Bart

**Comment by khorben** [comment]
Please read the patch again, the sections `{% if apache_app_listen_address.all is defined %}` are duplicated. I did not touch anything about HAProxy.

**Comment by khorben** [comment]
In the output from `git diff` it is not obvious, but the same sections (plus `include "ssl_backend.conf"`) are found further down in each file modified, within the same virtual host or section.

**Comment by quartje** [comment]
You're absolutely right, I misread the patch. Thanks for fixing this!
